### PR TITLE
teams fx samples migration

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

File: samples/tab-device-permissions/nodejs/appPackage/manifest.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   "version": "1.0.0",
-  "id": "7ae8f544-94b4-4a54-b5ce-25dd931c7a4a",
+  "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.microsoft.teams.extension",
   "developer": {
     "name": "Teams App, Inc.",
-    "websiteUrl": "https://<<BASE-URL>>",
-    "privacyUrl": "https://<<BASE-URL>>/privacy",
-    "termsOfUseUrl": "https://<<BASE-URL>>/termsofuse"
+    "websiteUrl": "https://www.microsoft.com",
```
Comment (formulahendry) [inline samples/tab-device-permissions/nodejs/appPackage/manifest.json:9]: same here. Is it by design to use www.microsoft.com, or better to use ${{TAB_DOMAIN}}?

File: samples/tab-personal-quickstart/ts/Manifest/manifest.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   "version": "1.0.0",
-  "id": "{{Microsoft-App-Id}}",
+  "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.microsoft.teams.extension",
   "developer": {
     "name": "Teams App, Inc.",
-    "websiteUrl": "https://{{domain-name}}",
-    "privacyUrl": "https://{{domain-name}}/privacy",
-    "termsOfUseUrl": "https://{{domain-name}}/termsofuse"
+    "websiteUrl": "https://www.microsoft.com",
```
Comment (formulahendry) [inline samples/tab-personal-quickstart/ts/Manifest/manifest.json:9]: same here. Is it by design to use www.microsoft.com, or better to use ${{TAB_DOMAIN}}?

File: samples/tab-channel-group-quickstart/ts/appPackage/manifest.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
   "version": "1.0.0",
-  "id": "c69434a5-fc52-471a-af42-ad5706de502f",
+  "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.microsoft.teams.extension",
   "developer": {
     "name": "Teams App, Inc.",
-    "websiteUrl": "https://localhost:3000",
-    "privacyUrl": "https://localhost:3000/privacy",
-    "termsOfUseUrl": "https://localhost:3000/termsofuse"
+    "websiteUrl": "https://www.microsoft.com",
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-channel-group-quickstart/ts/appPackage/manifest.json:9]: Fixed

File: samples/tab-device-permissions/nodejs/appPackage/manifest.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-device-permissions/nodejs/appPackage/manifest.json:9]: Fixed

File: samples/tab-personal-quickstart/ts/Manifest/manifest.json (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-personal-quickstart/ts/Manifest/manifest.json:9]: Fixed

**Review by formulahendry — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Please run build pipeline and share the updates.

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5644269266

image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving t.

**Pull request merged**